AR-Top-MobileApp/Assets/UserSettings.cs

```diff
@@ -37,12 +33,30 @@ public class UserSettings : MonoBehaviour {
 			PlayerPrefs.SetInt ("UserAA", 2);
 		}
 
-		if (PlayerPrefs.HasKey ("UserGrid")) {
-			onOutlineChanged(PlayerPrefs.GetString ("UserGrid"));
-		} else {
-			PlayerPrefs.SetString ("UserGrid", "Full");
-			onOutlineChanged ("Full");
+		setDropdownValue ();
+	}
+
+	void setDropdownValue (){
+		int index = 2;
+		if (PlayerPrefs.HasKey ("UserAA")) {
+			index = PlayerPrefs.GetInt ("UserAA");
 		}
+		TMP_Dropdown dropdown = dropdowns [0];
+		switch (index) {
```

Comment:
This switch case can probably be changed to a simple mathematical formula, we have an integer value that gets converted to another integer value? Just a thought.

Reply:
Changed it to a bitshift. Also changed the anti-aliasing setter to a powering function.